airflow/config_templates/default_airflow.cfg

```diff
@@ -1037,6 +1037,7 @@ task_track_started = True
 # :ref:`task duration<ui:task-duration>` from the task's :ref:`landing time<ui:landing-times>`.
 task_adoption_timeout = 600
 
+# Deprecated. Use scheduler.task_queued_timeout instead.
```

Comment:
Is this deprecation reflected in the code base by a DeprecationWarning?

Reply:
Updated to include DeprecationWarning

Reply:
we should remove it if no longer needed

Reply:
Would removing it unnecessarily break backward compatibility?